src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -297,11 +300,19 @@ public boolean reconnect()
 	}
 
 	/**
-	 * Simulates a Player consuming an Edible Item
+	 * Simulates a Player consuming an Edible Item. Some edibles inflict status effects on the consumer with a certain
+	 * probability.
 	 *
-	 * @param consumable The Item to consume
+	 * @param consumable          The Item to consume
+	 * @param inflictPotionEffect A flag determining whether potion effects that don't have a probability of 1 are
+	 *                            inflicted on the consumer. For example, consuming {@link Material#CHICKEN} has a 0.3
+	 *                            chance to inflict the hunger effect. If this flag is true, then the hunger effect will
+	 *                            be inflicted. Otherwise, it won't be inflicted. This does not prevent effects from
+	 *                            being inflicted that have a probability of 1. These will be inflicted regardless. The
+	 *                            flag is ignored if the edible does not inflict any potion effects.
```

Comment:
If possible, can this description be shortened? Maybe to something like this (you're probably better at english than me though)
```
Whether to always inflict potion effect from food regardless of probability. If this is is `false` and the food 
item has a probability to inflict the effect lesser than 0, it will not do so. 
```

Reply:
I shortened it a little, but I think adding a sentence that the flag does change the behaviour of effects with a probability of 1 as well as a sentence that the flag does not anything at all if the item does not inflict any status effects, is necessary clarification.